frontend/src/themes/typography.ts

```diff
@@ -41,6 +41,12 @@ const textStyles = {
     lineHeight: "23px",
     letterSpacing: "0",
   },
+  smallerParagraph: {
+    fontSize: "16px",
+    fontWeight: "400",
+    lineHeight: "20.83px",
+    letterSpacing: "0",
+  },
```

Comment:
Is this a new typography in the design system? If we are adding it in the `typography.ts` file, let's make sure to ping to keep the design system in up to date!